### PR TITLE
Revise package structure as discussed in google/openhtf issue #314.

### PR DESCRIPTION
**Pull request by jettisonjoe**

### CODE REVIEW

**Comment by jettisonjoe** [comment]
@grybmadsci @fahhem
Lining this PR up for review after station_api and pylint. Tests are failing right now due to args being parsed earlier in the import sequence before simulated command-line flags are inserted in the conf unittest. Looking at it, I'm not sure how this ever worked. Suggestions on how to fix that are welcome (in addition to overall feedback on package restructuring).

File: openhtf/util/conf.py [hunk 1]
```diff
@@ -166,8 +166,8 @@ def do_stuff():
 
 import mutablerecords
 
-from openhtf.util import argv
-from openhtf.util import threads
+import argv
```
Comment (grybmadsci) [inline openhtf/util/conf.py]: probably better to still refer to these absolutely as `from openhtf.util import argv` and threads.

**Comment by grybmadsci** [comment]
the weird arg injecting thing i did is not really the right way to do it.. you might look into updating the args we actually pass to the test in setup.py, where we exec pytest.

**Comment by jettisonjoe** [comment]
Moving to [new PR](https://github.com/google/openhtf/pull/411) for final review and merge.

**Pull request closed without merging**